cmd/pmem-csi-driver/main.go

```diff
@@ -66,12 +66,25 @@ func main() {
 		Version:            version,
 	})
 	if err != nil {
-		fmt.Printf("Failed to Initialized driver: %s", err.Error())
-		os.Exit(1)
+		errorexit("Failed to initialize driver:", err)
 	}
 
 	if err = driver.Run(); err != nil {
-		fmt.Printf("Failed to run driver: %s", err.Error())
-		os.Exit(1)
+		errorexit("Failed to run driver:", err)
 	}
 }
+
+func errorexit(msg string, e error) {
+	str := msg + e.Error()
+	fmt.Println(str)
+	terminationlogpath := os.Getenv("TERMINATION_LOG_PATH")
+	if terminationlogpath == "" {
+		terminationlogpath = os.Getenv("/dev/termination-log")
+	}
+	f, err := os.Create(terminationlogpath)
+	if err == nil {
+		f.WriteString(str)
+		f.Close()
+	}
```

Comment:
Using `ioutil.WriteFile` is simpler.

Reply:
I used os because it's already imported and operates on string, where ioutil.WriteFile needs new import and operates on byte[]. No big difference, we can indeed save few lines where we actually call it.

Reply:
Let's always keep the source code as simple as possible in cases where runtime overhead isn't relevant (which is almost always the case).